src/main/java/eu/dissco/backend/domain/MappingTerms.java

```diff
@@ -0,0 +1,70 @@
+package eu.dissco.backend.domain;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public enum MappingTerms {
+  COUNTRY("country", "digitalSpecimen.ods:attributes.dwc:country.keyword"),
+  COUNTRY_CODE("countryCode", "digitalSpecimen.ods:attributes.dwc:countryCode.keyword"),
+  MIDS_LEVEL("midsLevel", "midsLevel"),
+  PHYSICAL_SPECIMEN_ID("physicalSpecimenId", "digitalSpecimen.ods:physicalSpecimenId.keyword"),
+  TYPE_STATUS("typeStatus", "digitalSpecimen.ods:attributes.dwc:typeStatus.keyword"),
+  LICENSE("license", "digitalSpecimen.ods:attributes.dcterms:license.keyword"),
+  HAS_MEDIA("hasMedia", "digitalSpecimen.ods:attributes.ods:hasMedia.keyword"),
+  ORGANISATION_ID("organisationId", "digitalSpecimen.ods:attributes.ods:organisationId.keyword"),
+  ORGANISATION_NAME("organisationName",
+      "digitalSpecimen.ods:attributes.ods:organisationName.keyword"),
+  SOURCE_SYSTEM_ID("sourceSystemId", "digitalSpecimen.ods:attributes.ods:sourceSystemId.keyword"),
+  SPECIMEN_NAME("type", "digitalSpecimen.ods:type.keyword"),
+  DATASET_ID("datasetId", "digitalSpecimen.ods:attributes.ods:datasetId.keyword"),
+  QUERY("q", "q");
+
+  private final String name;
+  private final String fullName;
+
+  MappingTerms(String name, String fullName) {
+    this.name = name;
+    this.fullName = fullName;
+  }
+
+  public static Map<String, String> getParamMapping() {
```

Comment:
I don't think these should be functions. It doesn't make sense to re-construct the list/hash map on every call. Would be better to make these both static & final variables instead.